podcasts/PCViewController.swift

```diff
@@ -93,14 +99,15 @@ class PCViewController: SimpleNotificationsViewController {
     }
 
     @objc func refreshRightButtons() {
-        if supportsGoogleCast {
+        if supportsGoogleCast || !extraRightButtons.isEmpty {
```

Comment:
This logic seems wrong to me. Or maybe I'm missing something.

Given `supportsGoogleCast` is `false` but `extraRightButtons` is not empty. The Chromecast icon will be displayed even though it should not.

Reply:
Thanks for detecting this! Fix done!